app/src/androidTest/java/com/pronoidsoftware/auth/presentation/login/LoginScreenTest.kt

```diff
@@ -0,0 +1,46 @@
+package com.pronoidsoftware.auth.presentation.login
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.navigation.compose.rememberNavController
+import com.pronoidsoftware.core.TaskyAndroidTest
+import com.pronoidsoftware.core.presentation.designsystem.TaskyTheme
+import com.pronoidsoftware.tasky.MainActivity
+import com.pronoidsoftware.tasky.NavigationRoot
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Rule
+import org.junit.Test
+
+@HiltAndroidTest
+class LoginScreenTest : TaskyAndroidTest() {
+
+    @get:Rule
+    val composeRule = createAndroidComposeRule<MainActivity>()
+
+    @Test
+    fun testRegisterScreenUi_allValid() {
+        composeRule.activity.setContent {
```

Comment:
Doesn't this break because it would call `setContent` twice on the activity? 🤔

Reply:
Ah, good catch. It doesn't break, and as far as I can tell, it's the recommended way to set up the screen for your test. But I also think I'm doing away with this test in favor of a E2E test that will encapsulate this anyways 😅 One thing I did find with this test though is that because I was running the Robot inside of `setContent{}`, none of the actual assertions would run and the test would automatically pass!